modules/libevent/2.1.12-stable.bcr.0/overlay/event2-config_apple.h

```diff
@@ -0,0 +1,520 @@
+/* event-config_apple.h
+ *
+ * This file was generated by cmake but modified for Bazel.
+ *
+ * Do not rely on macros in this file existing in later versions.
+ */
+#ifndef EVENT2_CONFIG_APPLE_H_INCLUDED_
+#define EVENT2_CONFIG_APPLE_H_INCLUDED_
+
+/* Numeric representation of the version */
+#define EVENT__NUMERIC_VERSION 0x02010c00
+#define EVENT__PACKAGE_VERSION "2.1.12"
```

Comment:
Would it be possible to break out the version constants into a new file or template in `module_version()` via skylib's `expand_template` rule? It would increase the change that this file can be reused as is or even symlinked by later versions.

Reply:
I like it! Added a `event2-config_common.h` file.

I would question the idea of using a symlink instead of directly copying the file - that might be a premature optimization that is going to trip something up. I guess it isn't relevant yet though.